README.md

```diff
@@ -1,289 +1,101 @@
+<img src="https://wso2.cachefly.net/wso2/sites/all/image_resources/wso2-branding-logos/wso2-logo-orange.png" alt="WSO2 logo" width=30% height=30% />
+
 # WSO2 API Manager
-        
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![stackoverflow](https://img.shields.io/badge/Get%20Support%20on%20Stack%20Overflow-wso2am-orange)](https://stackoverflow.com/tags/wso2-am/)
+[![Join the community on Discord](https://img.shields.io/badge/Join%20us%20on%20Discord-wso2--apim-blueviolet)](https://discord.com/invite/wso2)
+[![X](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow%20Us)](https://twitter.com/intent/follow?screen_name=wso2)
 
 ---
 
+WSO2 API Manager is a complete platform for building, integrating, and exposing your digital services as managed APIs in the cloud, on-premise, and hybrid architectures to drive your digital transformation strategy.
 
-[![Build Status](https://wso2.org/jenkins/job/products/job/product-apim/badge/icon)](https://wso2.org/jenkins/job/products/job/product-apim/)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![stackoverflow](https://img.shields.io/badge/stackoverflow-wso2am-orange)](https://stackoverflow.com/tags/wso2-am/)
-[![slack](https://img.shields.io/badge/slack-wso2--apim-blueviolet)](https://join.slack.com/t/wso2-apim/shared_invite/enQtNzEzMzk5Njc5MzM0LTgwODI3NmQ1MjI0ZDQyMGNmZGI4ZjdkZmI1ZWZmMjNkY2E0NmY3ZmExYjkxYThjNzNkOTU2NWJmYzM4YzZiOWU?src=sidebar)
+It allows API developers to design, publish, and manage the lifecycle of APIs and API product managers to create API products from one or more APIs.
 
----
+To learn more about WSO2 API Manager please visit https://wso2.com/products/api-manager.
 
-WSO2 API Manager (WSO2 API-M) is a powerful platform for creating, managing, consuming, and
-monitoring web APIs. It combines tried and tested SOA best practices with modern
-day API management principles to solve a wide range of enterprise challenges
-associated with API provisioning, governance, and integration.
+> If you run your workloads on Kubernetes, we recommend you check our specific solution for the Kubernetes platform https://wso2.com/api-platform-for-k8s/ 
+>
 
-WSO2 API Manager consists of several loosely coupled modules.
+# Why WSO2 API Manager?
```

Comment:
Generally having only one level one heading (#) in a README.md file is considered a best practice. So all these secondary headings should be level 2, isn't it?

Reply:
Yes +1